prow/config.yaml

```diff
@@ -961,6 +961,40 @@ postsubmits:
       nodeSelector:
         cloud.google.com/gke-nodepool: build-pool
 
+  - name: proxy-postsubmit
+    agent: kubernetes
+    context: prow/proxy-postsubmit.sh
+    always_run: true
+    rerun_command: "/test proxy-postsubmit"
+    trigger: "(?m)^/(retest|test (all|proxy-postsubmit))?,?(\\s+|$)"
```

Comment:
I am not clear how the test will be triggered.  Does it always need a PR?  How about if we merge the changes directly without PR(we can do that since it is a private branch).